desktop/containerd/index.md

```diff
@@ -32,186 +32,100 @@ images** checkbox.
- => [linux/amd64 internal] load metadata for docker.io/library/alpine:latest                                                                                0.6s
- => [linux/amd64 1/1] FROM docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad                                 0.0s
- => => resolve docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad                                             0.0s
- => CACHED [linux/arm64 1/1] FROM docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad                          0.0s
- => => resolve docker.io/library/alpine@sha256:bc41182d7ef5ffc53a40b044e725193bc10142a1243f395ee852a8d9730fc2ad                                             0.0s
- => exporting to image                                                                                                                                      0.0s
- => => exporting layers                                                                                                                                     0.0s
- => => exporting manifest sha256:71bf02afcd7a791c268aa935027f1dc05238f5b5017d755d0fd6d9c71c1b79b9                                                           0.0s
- => => exporting config sha256:f1edbf6b99d22831f9312ab2b8b7642a904c614000bb8369ed673848f4f03578                                                             0.0s
- => => exporting manifest sha256:e41da7320a956163128d77ad69c8109af4799b41bd2e8e660bc6f01136b67f45                                                           0.0s
- => => exporting config sha256:4a5580ab8335432cf6cea7ff695f177d120fa2c9aa4002525025888e3cae16ee                                                             0.0s
- => => exporting manifest list sha256:339d089b539c950268de1edeef9652584e16efa51ea2c84ee586d3143b97234d                                                      0.0s
- => => naming to docker.io/<username>/hello-friends:latest                                                                                              0.0s
- => => unpacking to docker.io/<username>/hello-friends:latest
-
+$ docker buildx create --bootstrap
+[+] Building 2.4s (1/1) FINISHED
+ => [internal] booting buildkit
+ => => pulling image moby/buildkit:buildx-stable-1
+ => => creating container buildx_buildkit_objective_blackburn0
+objective_blackburn
+$ docker buildx build --quiet \
+  --platform=linux/amd64,linux/arm64 \
+  --builder=objective_blackburn \
+  --load .
+ERROR: docker exporter does not currently support exporting manifest lists
 ```
 
-Run multi-platform image:
+Enabling the containerd image store lets you build, and load, multi-platform images
+to your local image store, all while using the default builder.
 
 ```console
-$ docker run <username>/hello-friends
-
+$ docker info | grep "Storage Driver" -A1 
```

Comment:
Perhaps use `--format` (where possible, I try to avoid extra tools, as they may not be available on Windows);

```bash
$ docker info --format "{{.Driver}}"
stargz
```

or (if we want both);

```
$ docker info --format "{{.Driver}}"
stargz

$ docker info --format "{{.DriverStatus}}"
[[driver-type io.containerd.snapshotter.v1]]
```